src/templates/partials/lighttpd.hbs

```diff
@@ -11,36 +15,59 @@ $SERVER["socket"] == ":80" {
 {{/if}}
 }
 
+#server.modules += ("mod_setenv")
+$HTTP["scheme"] == "https" {
+    # HTTP Strict Transport Security ({{output.hstsMaxAge}} seconds)
+    setenv.add-response-header = (
+        "Strict-Transport-Security" => "max-age={{output.hstsMaxAge}}"
+    )
+}
 {{/if}}
+
+#server.modules += ("mod_openssl")
 $SERVER["socket"] == ":443" {
-    ssl.engine   = "enable"
+    ssl.engine  = "enable"
 
+  {{#if (minver "1.4.53" form.serverVersion)}}
+    ssl.privkey = "/path/to/private_key"
+    ssl.pemfile = "/path/to/signed_cert"
+    ssl.ca-file = "/path/to/intermediate_certificate"
+    # alternative: pemfile is cert+privkey, ca-file is the intermediate chain in one file
+    #ssl.pemfile = "/path/to/signed_cert_plus_private_key"
+    #ssl.ca-file = "/path/to/intermediate_certificate"
```

Comment:
We try to be opinionated, just pick one option and present it to them.

Reply:
ok, updated